gradle/publish-jar.gradle

```diff
@@ -0,0 +1,24 @@
+apply plugin: 'com.jfrog.bintray'
```

Comment:
@iamdanfox, maybe worth changing this `publish-jar.gradle` to `publish-jar-base.gradle` if people want to customize it?

Reply:
Nah I'd prefer to try to find a common thing that's good enough for all our OSS repos so we don't have some of them customizing and some of them not customizing!